packages/create-next-app/package.json

```diff
@@ -47,5 +47,8 @@
     "typescript": "3.8.3",
     "update-check": "1.5.4",
     "validate-npm-package-name": "3.0.0"
+  },
+  "engines": {
+    "node": ">=10.13.0"
```

Comment:
This has been updated recently

https://github.com/vercel/next.js/blob/beedff291eb6e5f6308eddd348bd24a62a3973e4/packages/create-next-app/package.json#L49-L51